store/nodegroup.go

```diff
@@ -70,6 +79,15 @@ func (ng *NodeGroup) Update(name, endpoint string, platforms []string, endpoints
 		if flags != nil {
 			n.Flags = flags
 		}
+		if do != nil {
+			n.DriverOpts = do
+		}
+		if configFile != "" {
+			for k, v := range files {
+				n.Files[k] = v
+			}
+		}
+		logrus.Warn("new settings may not be used until builder is restarted")
```

Comment:
Only warn if there was a change in one of the fields that require restart (I believe all if conditions except the platform one).